ckb-importer/src/main/java/com/hartwig/serve/ckb/classification/CkbClassificationConfig.java

```diff
@@ -300,6 +304,32 @@ private static Set<String> ebvPositiveEvents() {
         return Sets.newHashSet();
     }
 
+    @NotNull
+    private static Set<String> absenceOfProteinEvents() {
+        return Sets.newHashSet("negative");
+    }
+
+    @NotNull
+    private static Set<String> absenceOfProteinBlacklistEvents() {
+        Set<String> set = Sets.newHashSet();
+        set.add("HRD");
+        set.add("MSI");
+        return set;
+    }
+
+    @NotNull
+    private static Set<String> presenceOfProteinEvents() {
+        return Sets.newHashSet("positive");
+    }
+
+    @NotNull
+    private static Set<String> presenceOfProteinBlacklistEvents() {
+        Set<String> set = Sets.newHashSet();
+        set.add("HRD");
+        set.add("MSI");
+        return set;
+    }
+
```

Comment:
I would order these functions in the same ordering as how the fields are set themselves.